packages/kobotoolbox/src/Utils.js

```diff
@@ -15,7 +15,18 @@ export const prepareNextState = (state, response) => {
 };
 
 export async function request(state, method, path, opts) {
-  const { baseURL, apiVersion, username, password } = state.configuration;
+  const { baseUrl, apiVersion, username, password } = state.configuration;
+
+  let baseURL;
+
+  if (state.configuration.baseURL) {
```

Comment:
Ok but this way, if baseURL and baseUrl are set, baseURL will win AND users will get an incorrect warning.

I find it kinda weird too that the code uses the wrong styling of `baseURL`. It would feel cleaner if code and config were consistent and baseURL was the outlier.

So:
`baseUrl` needs to be the main variable
`if (!baseURL && state.configuration.baseURL) { /* warn and assign */ }

Reply:
Please don't rush your solution! It is very expensive to keep revisiting little fixes like this